src/app/component/MyAccountMyOrders/MyAccountMyOrders.component.js

```diff
@@ -13,15 +13,251 @@ import { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import './MyAccountMyOrders.style';
 
+import TextPlaceholder from 'Component/TextPlaceholder';
+import { Link } from 'react-router-dom';
+import { formatCurrency } from 'Util/Price';
+
+export const ORDERS_PATH = 'orders-list';
+
+const MY_ACCOUNT_PREFIX = 'my-account';
+const CREATED_AT = 'created_at';
+const GRAND_TOTAL = 'grand_total';
+const ID = 'increment_id';
+const STATUS_LABEL = 'status_label';
+const SHOW_MORE = 'show_more';
+
 class MyAccountMyOrders extends PureComponent {
     static propTypes = {
-        // TODO: implement prop-types
+        getOrderById: PropTypes.func.isRequired,
+        getOrderList: PropTypes.func.isRequired,
+        getFormattedDate: PropTypes.func.isRequired,
+        orderId: PropTypes.oneOfType([
+            PropTypes.string, PropTypes.number
+        ]).isRequired,
+        orderList: PropTypes.arrayOf(PropTypes.shape).isRequired,
```

Comment:
Why are you using shape without an argument, this is wrong. You must define an order type